discord/webhook.py

```diff
@@ -451,17 +462,18 @@ def __init__(self, data, *, adapter, state=None):
     def __repr__(self):
         return '<Webhook id=%r>' % self.id
 
+    def __str__(self):
+        return self.name
```

Comment:
If `self.name` is `None` (which is possible) then this will error out. Best to not provide this.